features/facebookImport/src/components/dataOverviewMiniStory/dataOverviewMiniStory.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import DataBubblesAll from "../dataViz/dataBubblesAll.jsx";
+
+import i18n from "../../i18n.js";
+
+import "./dataOverviewMiniStory.css";
+
+const DataOverviewMiniStory = ({ data }) => {
+    data.sort(function (a, b) {
+        return b.count - a.count;
+    });
+
+    const dataBubblesWidth = 400;
+    const dataBubblesHeight = 400;
+    const dataBubblesDarkColor = "#0f1938";
+    const dataBubblesLightColor = "#f7fafc";
+    const [clickedButton, setClickedButton] = useState(data[0].title);
+
+    const onClickButton = (ev, newClickedButton) => {
+        ev.preventDefault();
+        setClickedButton(newClickedButton);
+    };
+
+    const bubbleColor = (bubbles) => {
+        if (bubbles.data.title === clickedButton) {
+            return dataBubblesLightColor;
+        } else {
+            return dataBubblesDarkColor;
+        }
+    };
+
+    let fileNumber = data[0].count;
+    data.map((bubble) => {
+        bubble.title === clickedButton
+            ? (fileNumber = bubble.count)
+            : fileNumber;
+    });
```

Comment:
That logic seems a bit odd - you're just trying to set `fileNumber` to `bubble.count` where `bubble.title === clickedButton`, right? Should be doable like this:

```
const fileNumber =
    data.find((bubble) => bubble.title === clickedButton)?.count
    ?: data[0].count;
```